modules/storages/app/services/storages/storages/set_attributes_service.rb

```diff
@@ -54,5 +76,9 @@ def derive_default_storage_name
 
       "#{prefix} #{last_id + 1}"
     end
+
+    def nextcloud_storage?
+      storage.provider_type_nextcloud? && storage.is_a?(Storages::NextcloudStorage)
```

Comment:
Noob question: Is there a way that the second can be true if the first isn't? 😕

Reply:
Good eye, I think we can just do with the later 👍🏾